installing/installing_ibm_power/installing-ibm-power.adoc

```diff
@@ -0,0 +1,91 @@
+[id="installing-ibm-power"]
+= Installing a cluster on IBM Power
+include::modules/common-attributes.adoc[]
+:context: installing-ibm-power
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+IBM Power infrastructure that you provision.
+
+[IMPORTANT]
+====
+Additional considerations exist for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms] before you install an {product-title} cluster.
+====
+
+.Prerequisites
+
+* Provision
+xref:../../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storage-nfs[persistent storage using NFS]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.hcpa6/bhslzvs.htm[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in the IBM Knowledge Center.
+
+include::modules/installation-infrastructure-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-network-user-infra.adoc[leveloffset=+2]
+
+include::modules/installation-dns-user-infra.adoc[leveloffset=+2]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+1]
+
+include::modules/installation-bare-metal-config-yaml.adoc[leveloffset=+2]
+
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+1]
+
+[id="creating-machines-bare-metal-power"]
+== Creating {op-system-first} machines
+
+Before you install a cluster on bare metal infrastructure that you provision,
```

Comment:
Should this say something other than bare metal, or is this correct?

Reply:
Should be IBM Power, fixed.